build.sbt

```diff
@@ -4,7 +4,22 @@ licenses += ("MIT", url("http://opensource.org/licenses/MIT"))
 homepage := Some(url("http://github.com/typelevel/kind-projector"))
 
 scalaVersion := "2.12.8"
-crossScalaVersions := Seq("2.10.7", "2.11.12", "2.12.8", "2.13.0")
+crossScalaVersions := Seq("2.10.7", "2.11.12", "2.12.8", "2.12.9", "2.12.10", "2.13.0", "2.13.1")
```

Comment:
With the proliferation of Scala versions in here, it makes sense to use https://github.com/dwijnand/sbt-travisci.

Reply:
Are you suggesting I add this in this PR?

Reply:
Yes, that would be great!